lib/omniship/newgistics/track/package.rb

```diff
@@ -1,29 +1,14 @@
 module Omniship
   module Newgistics
     module Track
-      class Package
+      class Package < Omniship::Base
+        ARRIVAL_STATUSES = ["Delivered"]
+        DEPARTURE_STATUSES = ["Received", "InTransit", "Departed", "InUSPSNetwork"] + ARRIVAL_STATUSES
```

Comment:
I asked because this is the only place you did that. For other methods, you have arrival/departure separated.